sktime/param_est/lag.py

```diff
@@ -74,12 +74,13 @@ class ARLagOrderSelector(BaseParamFitter):
     >>> from sktime.param_est.lag import ARLagOrderSelector
     >>> y = load_airline()
     >>> selector = ARLagOrderSelector(maxlag=12, ic="bic")
-    >>> selector.fit(y)
+    >>> selector.fit(y)  # doctest: +ELLIPSIS
     ARLagOrderSelector(...)
-    >>> selector.selected_model_
-    (3,)
-    >>> selector.ic_value_
-    1369.6963340649502
+    >>> isinstance(selector.selected_model_, (list, tuple))
```

Comment:
please remove this, the doctest is also displayed as an example in the docs. "isinstance" is a typical test clause, it does not help the user.